feature/search/src/test/kotlin/uk/govuk/app/search/SearchViewModelTest.kt

```diff
@@ -24,14 +30,37 @@ class SearchViewModelTest {
         }
     }
 
+    @OptIn(ExperimentalCoroutinesApi::class)
```

Comment:
You can add this annotation at class level to save you adding it to every test function.